open-metadata-implementation/adapters/open-connectors/repository-services-connectors/open-metadata-collection-store-connectors/omrs-rest-repository-connector/pom.xml

```diff
@@ -39,6 +39,11 @@
             <artifactId>open-connector-framework</artifactId>
          </dependency>
 
+        <dependency>
```

Comment:
This doesn;t seem to be needed, and is what is causing the PR compile to fail. spring-web is not used in the java code so should not be a dependency

Reply:
Hi, I fixed this PR, the issue is that locally it is complaining about this dependency. I have deleted .m2 repo, but the issue persists. Any other suggestion I could try for my local build?

Reply:
Hk Nigel, this is the dependency tree:
[INFO] --- maven-dependency-plugin:3.1.1:tree (default-cli) @ omrs-rest-repository-connector 
[INFO] org.odpi.egeria:omrs-rest-repository-connector:jar:1.1-SNAPSHOT
[INFO] +- org.odpi.egeria:rest-client-factory:jar:1.1-SNAPSHOT:compile
[INFO] |  +- org.slf4j:slf4j-api:jar:1.7.26:compile
[INFO] |  \- org.odpi.egeria:spring-rest-client-connector:jar:1.1-SNAPSHOT:compile
[INFO] |     +- org.springframework:spring-web:jar:5.1.8.RELEASE:compile

Reply:
Issue is fixed locally, thanks for help!